exopy_qm/tasks/tasks/ConfigureExecuteTask.py

```diff
@@ -0,0 +1,264 @@
+import importlib
+import importlib.util
+import ast
+
+from exopy.tasks.api import InstrumentTask
+from atom.api import Unicode, Long, Float, Value, Typed
```

Comment:
Please don't use `Long` it is (or should be deprecated) and under Python 3 is fully equivalent to `Int`.